From 0cf08954625e1ea9e9646d0c009925460e7fb3ec Mon Sep 17 00:00:00 2001
From: Tobias Schramm <t.schramm@t-sys.eu>
Date: Sat, 15 May 2021 21:44:05 +0200
Subject: [PATCH 7/7] arm: sun8i: OtterCast Amp: add ethernet0 alias to emac

The board setup code uses the alias to generate MAC addresses that are
then exported into the environment.
This enables generation of the fist MAC address, which is placed into the
env var `ethaddr`.
---
 arch/arm/dts/sun8i-s3-ottercast-amp.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/dts/sun8i-s3-ottercast-amp.dts b/arch/arm/dts/sun8i-s3-ottercast-amp.dts
index ed86554f30..d8a20e9f14 100644
--- a/arch/arm/dts/sun8i-s3-ottercast-amp.dts
+++ b/arch/arm/dts/sun8i-s3-ottercast-amp.dts
@@ -14,6 +14,7 @@
 
 	aliases {
 		serial0 = &uart1;
+		ethernet0 = &emac;
 	};
 
 	chosen {
-- 
2.31.1

